GrandCentralBoard/Core/Widget.swift

```diff
@@ -6,16 +6,38 @@
 import UIKit
 
 
-public protocol Updateable : class {
+/// Class implementing this protocol will have a collection of sources implementing `UpdatingSource` protocol.
+public protocol HavingSources : class {
     var sources: [UpdatingSource] { get }
+}
+
+/// Ability to be updated by calling `update(source:)` method with a source implementing `UpdatingSource` protocol.
+public protocol Updateable : class {
     func update(source: UpdatingSource)
 }
 
-public protocol Widget: Updateable {
+/**
+  Class implementing this protocol have it's controlled `view` and has to implement `Updateable` and `HavingSources` protocols.
+ */
+public protocol WidgetControlling: Updateable, HavingSources {
```

Comment:
I'm pretty sure about those suffixes are a great thing. They *force* you to think about protocols as abilities.

The class implementing `WidgetControlling` is de facto widget controller (and name change is planned https://github.com/macoscope/GrandCentralBoard/issues/21). 

It holds a view like `ViewController` does.

More philosophical question: What IS a widget? It's a view? I would say `Widget` is a triangle: `View` + `Controller` + `Source`. Naming one class `Widget` is not good.